File: web-app/src/app/services/types/Collegejobposting.ts
import { Campusregistration } from "./Campusregistration";
import { Jobposting } from "./Jobposting";

export type Collegejobposting = {
    Id : number;
    JobPostingId?: number;
    CollegeId?: number;
    College : Campusregistration;
    JobPosting: Jobposting;
}

File: Placements.WebApi/Placement/Models/Collegejobposting.cs
using System;
using System.Collections.Generic;

namespace Placements.WebApi.Placement.Models;

public partial class Collegejobposting
{
    public long Id { get; set; }

    public long? JobPostingId { get; set; }

    public long? CollegeId { get; set; }

    public virtual Campusregistration? College { get; set; }

    public virtual Jobposting? JobPosting { get; set; }
}
